src/simplewallet/simplewallet.cpp

```diff
@@ -3257,7 +3256,7 @@ static bool might_be_partial_seed(const epee::wipeable_string &words)
   std::vector<epee::wipeable_string> seed;
 
   words.split(seed);
-  return seed.size() < 24;
+  return seed.size() < 25;
```

Comment:
I think we should revert. We shouldn't account for the date if the user is inputting seed words one by one because then there's no real improvement over just inputting your date in the second step when it prompts you.